src/variableProcessor.ts

```diff
@@ -48,6 +54,23 @@ export class VariableProcessor {
             }
         }
 
+        let requestVariables = VariableProcessor.getRequestVariablesInCurrentFile();
+        for (let [variableName, variableValue] of requestVariables) {
+            let regex = new RegExp(`\\{\\{\\s*${variableName}.*\\s*\\}\\}`, 'g');
+            let matches = request.match(regex);
+            if (matches && matches.length > 0) {
+                for (var i = 0; i < matches.length; i++) {
+                    var requestVariable = matches[i].replace('{{', '').replace('}}', '');
+                    try {
+                        const value = RequestVariableCacheValueProcessor.getValueAtPath(variableValue, requestVariable);
+                        request = request.replace(new RegExp(`\\{\\{\\s*${requestVariable}\\s*\\}\\}`, 'g'), value.toString());
+                    } catch {

```

Comment:
Does catch without exception work in your environment? I thought JS needs us to provide an error variable even if we don't need it like
```js
catch (e) {
...
}
```

According to this [doc](https://docs.microsoft.com/en-us/scripting/javascript/reference/try-dot-dot-dot-catch-dot-dot-dot-finally-statement-javascript), it explicitly said it's required

Reply:
Well, it works in my dev environment because I normally work with TypeScript 2.5+, which actually [does support optional catch clauses](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#typescript-25)

I see this repository depends upon `"typescript": "^2.0.3"`, but the caret `^` symbol there means that it will match any `2.x.x` release, up till but not including `3.0.0`. So that's probably why it works on my machine.

I'd recommend upgrading the `typescript` version anyway, there's a lot of nice stuff in the newer version. Also recommend locking down the actual version instead of using carets `^` or tilde `~`.

Reply:
@cbrevik Nice, I haven't known this before, nice, please ignore this comment.